examples/specialty_plots/radar_chart.py

```diff
@@ -38,14 +41,14 @@ def radar_factory(num_vars, frame='circle'):
     # calculate evenly-spaced axis angles
     theta = np.linspace(0, 2*np.pi, num_vars, endpoint=False)
 
+    # The Axes patch must be centered at (0.5, 0.5) and of radius 0.5 in axes
+    # coordinates.
+
     def draw_poly_patch(self):
-        # rotate theta such that the first axis is at the top
-        verts = unit_poly_verts(theta + np.pi / 2)
-        return plt.Polygon(verts, closed=True, edgecolor='k')
+        return RegularPolygon((0.5, 0.5), num_vars, radius=.5, edgecolor="k")
 
     def draw_circle_patch(self):
```

Comment:
Since `draw_poly_patch` and `dras_circle_patch` just return simple patches now, you can simplify all the overhead and create the appropriate patch directly in `_gen_axes_patch`.